src/gui/KM_InterfaceGame.pas

```diff
@@ -149,6 +158,12 @@ constructor TKMUserInterfaceGame.Create(aRender: TRender);
 
   fMinimap := TKMMinimap.Create(False, False);
   fViewport := TKMViewport.Create(aRender.ScreenX, aRender.ScreenY);
+
+  fDragScrolling := False;
+  fDragScrollingCursorPos.X := 0;
+  fDragScrollingCursorPos.Y := 0;
+  fDragScrollingViewportPos := ZERO_POINT_F;
```

Comment:
I know, but anyway I prefer to explicitly set them. So when looking through the code (with Ctrl+Shift+Enter f.e.) I can see its initialization value.

Here I just replaced 

`fDragScrollingViewportPos.X := 0;
fDragScrollingViewportPos.Y := 0;`
with 
`fDragScrollingViewportPos := ZERO_POINT_F;`

ALso I was planning to do small refactoring: replace all KMPoint(0, 0) and KMPointF(0.0, 0.0) with new constants. Don't you mind it ?